requirements.txt

```diff
@@ -11,7 +11,8 @@ GitPython>=3.1.42
 httpx>=0.25.0
 instructlab-quantize>=0.1.0
 instructlab-schema>=0.2.0
-instructlab-sdg>=0.0.4
+#instructlab-sdg>=0.0.4
+instructlab-sdg @ git+https://github.com/russellb/sdg.git@new-cli-integration
```

Comment:
You are probably installing from git for testing.
Just for the record, PyPI does not accept packages with these types of links. Our CI doesn't detect the issue, yet.

Reply:
Thanks!  Yeah, this was just for testing. That branch merged into the repo, so it could chain to the instructlab/sdg repo and main branch, but it's still not in a release.

I posted it here for early feedback and testing on the approach